contributors/devel/cherry-picks.md

```diff
@@ -25,17 +25,19 @@ task is for backporting PRs from master to release branches.
    `upstream/release-3.14`: `hack/cherry_pick_pull.sh upstream/release-3.14
    98765`
  * Your cherrypick PR targeted to the release branch will immediately get the
-   `do-not-merge/cherry-pick-not-approved` label. The release branch owner
+   `do-not-merge/cherry-pick-not-approved` label. The Patch Release Manager
```

Comment:
nit: link Patch Release Manager to https://github.com/kubernetes/sig-release/blob/master/release-team/README.md#patch-release-manager

Reply:
ahhhhh yes this is what I was looking for. Thank you!